pyprep/find_noisy_channels.py

```diff
@@ -90,6 +79,25 @@ def __init__(self, raw, do_detrend=True, random_state=None, matlab_strict=False)
         self.bad_by_dropout = []
         self.bad_by_ransac = []
 
+        # Get original EEG channel names, channel count & samples
+        ch_names = np.asarray(self.raw_mne.info["ch_names"])
+        self.ch_names_original = ch_names
+        self.n_chans_original = len(ch_names)
+        self.n_samples = raw._data.shape[1]
+
+        # Before anything else, flag bad-by-NaNs and bad-by-flats
+        self.find_bad_by_nan_flat()
+        bads_by_nan_flat = self.bad_by_nan + self.bad_by_flat
+
+        # Make a subset of the data containing only usable EEG channels
+        self.usable_idx = np.isin(ch_names, bads_by_nan_flat) == False
```

Comment:
```suggestion
        self.usable_idx = np.isin(ch_names, bads_by_nan_flat).astype(int) == 0
```

Reply:
Apparently `np.isin` has an `inverse` option that does what we want, I just missed it the first time around. Should be fixed now!

Reply:
much nicer! thanks @a-hurst